File: Matnas/Common/Volunteer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Common
{
   public  class Volunteer
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Address { get; set; }
        public string Telephone { get; set; }
        public string Pelephone { get; set; }
        public string Email { get; set; }
        public System.DateTime? Age { get; set; }
        public string Comments { get; set; }
        public bool? IsActive { get; set; }

    }
}

File: rmatnas/src/app/Classes/Volunteer.ts
export class Volunteer {
    Id: number;
    Name: string;
    Telephone: string;
    Pelephone: string;
    Address: string;
    Email: string;
    Age: string;
    Comments: string;
    IsActive: boolean;
    show: boolean;
    // tslint:disable-next-line: max-line-length
    constructor(Name: string, telephone: string, pelephone: string, email: string, address: string, birthdate: string, isActive: boolean) {
        this.Name = Name;
        this.Telephone = telephone;
        this.Pelephone = pelephone;
        this.Address = address;
        this.Email = email;
        this.Age = birthdate;
        this.IsActive = isActive;
        this.show = false;
    }

    public CalculateAge() {
        const birthday = new Date(this.Age);
        const timeDiff = Math.abs(Date.now() - birthday.getTime());
        return Math.floor((timeDiff / (1000 * 3600 * 24)) / 365.25);
    }
}
